readme.md

```diff
@@ -48,35 +55,45 @@ Life Cycle of a request/response in Rails:
 
 4. Once the request hits the controller, its then going to query the database through Active Record(the model) for the information specified in the controller.
 
-5. Once the controller has the information from the model that it needs it sends it to the view
+5. Once the controller has the information from the model that it needs, it sends it to the view
+
+6. The view takes the objects from the controller, and sends a response to the user.
 
-6. The view takes the objects from the controller and sends a response to the user.
+Let's dive into an example, and see MVC in action by writing some Rails code.
 
-Let's dive into an example, and see MVC in action by writing some rails code.
+## Set-Up (5 / 30)
 
-## Set-Up (5/30)
-This lesson will build directly off of the work you did with Models and Migrations.
+This lesson will build off of the completed work you will do in the Models and Migrations lesson, so just know we will not be going in depth into the database migrations or have to write any new Model code.
 
-If you would prefer to work with a clean starter rather than building on your existing copy:
+To get started, clone down the following repo:
```

Comment:
"...and if you have errors, Slack the back!" I wouldn't take time to troubleshoot in front of the class.